Bottom: adadcdd25320eb90ca724a7e2daff39b76e997e4
Top:    25b3606d38e4b6bcfd1d010b1b428c91cf9b189f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-23 18:53:40 +0100

parse_y.y: Remove unused GRIDFIT macro


---

diff --git a/src/parse_y.y b/src/parse_y.y
index 8eaba94..f35d9d0 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -42,7 +42,6 @@
 #include "config.h"
 #endif
 
-#define GRIDFIT(x,g) (int)(0.5 + (int)(((x)+(g)/2.)/(g))*(g))
 #include "global.h"
 #include "create.h"
 #include "data.h"
